GliaWidgets/Sources/View/Chat/Message/Content/File/Image/ChatImageFileContentView.swift

```diff
@@ -86,4 +88,20 @@ class ChatImageFileContentView: ChatFileContentView {
     private func setImage(_ image: UIImage?) {
         imageView.image = image
     }
+
+    private func updateAccessibilityIdentifier(with download: FileDownload) {
+        switch download.state.value {
+        case .none:
```

Comment:
This might be simplified by just appending the case's string value to the name. However then you would have `none` instead of `download`. But it might be worth it to have less code. However it's up to you, so I will approve.

Reply:
Fixed 7a437b567854a714f59962bf1f613ae32cb11dd4